projects/plugins/crm/src/event-manager/class-events-manager.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * Events_Manager class.
+ *
+ * @package Automattic\Jetpack\CRM\Event_Manager
+ */
+
+namespace Automattic\Jetpack\CRM\Event_Manager;
+
+/**
+ * Events_Manager class.
+ *
+ * @since $$next-version$$
+ */
+class Events_Manager {
+
+	/** @var null The Events_Manager instance. */
```

Comment:
Same comment as here: https://github.com/Automattic/jetpack/pull/32185#discussion_r1281131513
I think we should make everything WP coding standards compliant.

Reply:
Okay

Reply:
Fixed here: https://github.com/Automattic/jetpack/pull/32139/commits/9cf34431dd081cc89a1bda3b4cd991b1760a5c23